app/views/projects/index.html.erb

```diff
@@ -4,8 +4,7 @@
       <ul>
         <%= image_tag team_member.thumbnail %>
         <li><%= team_member.name %></li>
-        <li><%= team_member.role %></li>
+        <li><%= team_member.discipline %></li>
       </ul>
     <% end %>
```

Comment:
Indentation's a little weird in here now, but I think this is now in conflict with what's on `master`. You probably want to fetch and rebase on top of `origin/master`.